src/xcode/ENA/ENA/Source/Scenes/HealthCertificates/Overview/Cells/HealthCertifiedPerson/HealthCertifiedPersonTableViewCell.swift

```diff
@@ -206,6 +212,12 @@ class HealthCertifiedPersonTableViewCell: UITableViewCell, ReuseIdentifierProvid
 				bottomView.trailingAnchor.constraint(equalTo: cardView.trailingAnchor),
 				bottomView.bottomAnchor.constraint(equalTo: cardView.bottomAnchor),
 
+				// placement is based on new figma design
+				accessoryIconView.widthAnchor.constraint(equalToConstant: 12.0),
+				accessoryIconView.heightAnchor.constraint(equalToConstant: 21.0),
+				accessoryIconView.topAnchor.constraint(equalTo: gradientView.topAnchor, constant: 35.0),
+				accessoryIconView.trailingAnchor.constraint(equalTo: gradientView.trailingAnchor, constant: -18.0),
```

Comment:
We need a constraint between the `stackView.trailingAnchor` and the `accessoryIconView.leadingAnchor` as well so they don't overlap on bigger font sizes/smaller screen sizes.